### PR TITLE
snowling

### PR DESCRIPTION
**Pull request by SonixApache**
only rolls on snaxi
~~during testing my candidates keep getting trimmed out, not sure if this'll happen on live as well, i'm doing some more testing~~ fixed, probably

🆑 
 - experiment: Somebody in this snowmap ain't what he appears to be.

### CODE REVIEW

File: code/datums/gamemode/dynamic/dynamic_rulesets_roundstart.dm [hunk 1]
```diff
@@ -50,18 +50,21 @@
...
+
 /datum/dynamic_ruleset/roundstart/changeling
 	name = "Changelings"
 	role_category = CHANGELING
-	protected_from_jobs = list("Security Officer", "Warden", "Merchant", "Head of Personnel", "Detective", "Head of Security", "Captain")
-	restricted_from_jobs = list("AI","Cyborg","Mobile MMI")
-	enemy_jobs = list("Security Officer","Detective","Head of Security", "Captain")
-	required_enemies = list(1,1,0,0,0,0,0,0,0,0)
+	protected_from_jobs = list("Security Officer", "Warden","Merchant", "Head of Personnel", "Detective", "Head of Security", "Captain")
+	restricted_from_jobs = list("AI","Cyborg","Mobile MMI", "Chaplain")
```
Comment (jknpj) [inline code/datums/gamemode/dynamic/dynamic_rulesets_roundstart.dm]: :thinking: what the actual fuck

File: code/datums/gamemode/dynamic/dynamic_rulesets_roundstart.dm [hunk 2]
```diff
@@ -50,18 +50,21 @@
...
+	enemy_jobs = list("Security Officer","Detective", "Warden", "Head of Security", "Captain", "Chaplain")
```
Comment (jknpj) [inline code/datums/gamemode/dynamic/dynamic_rulesets_roundstart.dm]: :thinking: what the actual fuck

File: code/datums/gamemode/dynamic/dynamic_rulesets_roundstart.dm (hunk 1 above)
Comment (ShiftyRail) [reply code/datums/gamemode/dynamic/dynamic_rulesets_roundstart.dm]: G*d wouldn't let His chaplain be impersonated

**Comment by ShiftyRail** [comment]
I must ask why

**Comment by SonixApache** [comment]
@ShiftyRail because thematic

**Comment by Durgen** [comment]
based

**Comment by ShiftyRail** [comment]
Shitty map = shitty mode ?

A bold decision

**Comment by SonixApache** [comment]
@ShiftyRail shitty opinion = shitty post

ling was in the snow, this is the snow, thus this gets the ling
it fits

**Comment by plosky1** [comment]
@ShiftyRail Imagine hating fun this much

**Comment by Allspice** [comment]
wasn't "it's thematic" about 20% of the reason ling wasn't removed for X years

**Comment by DamianX** [comment]
xD

**Comment by ShiftyRail** [comment]
https://www.youtube.com/watch?v=Esy-776wcIo

File: maps/_map.dm [hunk 3]
```diff
@@ -107,6 +107,9 @@
 			load_dungeon(T)
 
 /datum/map/proc/map_ruleset(var/datum/dynamic_ruleset/DR)
+	if(istype(DR.role_category,/datum/role/changeling))
```
Comment (ShiftyRail) [inline maps/_map.dm:110]: I'm retarded
This always fails

Comment (SonixApache) [reply maps/_map.dm:110]: isn't that the point?

**Pull request merged**